7.GoogleCloudPlatform_MicroServices_Demo/README.md

```diff
@@ -0,0 +1,193 @@
+# GoogleCloudPlatform-MicroServices-Demo with Kubevela
+
+[MicroServices-Demo](https://github.com/GoogleCloudPlatform/microservices-demo) is a cloud-native microservices demo application.
+This Demo create a **Online Boutique** application. **Online Boutique** consists of a 10-tier microservices application. 
+The application is a web-based e-commerce app where users can browse items, add them to the cart, and purchase them. 
+In this example, we will deploy the demo in the local k8s cluster **kind**. 
+
+## Prerequisites
+* [istio](https://istio.io/)
```

Comment:
Explain why istio was list here? Do you mean install istio? Then what version is required?